src/vmshepherd/http/rpc_api.py

```diff
@@ -57,6 +57,31 @@ def rpc_logger(func):
         presets = await self.request.app.vmshepherd.preset_manager.list_presets()
         return list(presets.keys())
 
+    @enabled_checker
+    async def list_presets_with_tags(self):
+        """ Gets more verbose list of available presets
+
+        :return:  (dict of presets with tags)
+
+        :rtype: dict of dicts
```

Comment:
If you already took effort to specify return type why not properly type annotating the whole thing? 

Someone already started type annotating in this codebase before, like [here](https://github.com/DreamLab/VmShepherd/blob/master/src/vmshepherd/presets/preset.py#L26) - maybe we could consider making it a common practice :)

I'm willing to make a PR that would type annotate the rest

Reply:
Its good suggestion, but its about changing code in a lot of places, so in next PR, your, my or somebody :)